Cargo.lock

```diff
@@ -3889,9 +3890,9 @@ checksum = "fed34cd105917e91daa4da6b3728c47b068749d6a62c59811f06ed2ac71d9da7"
 
 [[package]]
 name = "futures"
-version = "0.3.21"
+version = "0.3.23"
```

Comment:
This is the troublesome change in Cargo.lock, right? So we expect unit test failures?